src/AutoRest.CSharp/Mgmt/Output/MgmtReferenceType.cs

```diff
@@ -16,5 +16,7 @@ public MgmtReferenceType(ObjectSchema objectSchema, BuildContext<MgmtOutputLibra
             : base(objectSchema, context)
         {
         }
+
+        public override bool IncludeConverter => ObjectSchema.Extensions?.MgmtPropertyReferenceType == true && ObjectSchema.Extensions?.MgmtReferenceType != true || base.IncludeConverter;
```

Comment:
(PropRefType && !RefType) || base.IncludeConverter. Lets add () to simplify the expression order.